patches/minecraft/net/minecraft/item/ItemElytra.java.patch

```diff
@@ -0,0 +1,18 @@
+--- ../src-base/minecraft/net/minecraft/item/ItemElytra.java
++++ ../src-work/minecraft/net/minecraft/item/ItemElytra.java
+@@ -61,4 +61,15 @@
+             return new ActionResult<ItemStack>(EnumActionResult.FAIL, itemstack);
+         }
+     }
++
++    /* ======================================== FORGE START =====================================*/
++    private static final net.minecraft.entity.ai.attributes.AttributeModifier MODIFIER = new net.minecraft.entity.ai.attributes.AttributeModifier(java.util.UUID.fromString("D41556AA-466D-46E3-8071-262C78065B6A"), "Elytra modifier", 1.0D, 0);
```

Comment:
This UUID should probably be made its own `public static final` reference, so that it doesn't need to be copied by users.

Reply:
This UUID should *not* be used elsewhere, as this is specifically the uuid the Elytra provides.